### PR TITLE
chore(package): remove `isobject` dependency (`dependencies`)

### PR DESCRIPTION
**Pull request by hibiyasleep**
isobject
--------
does *really* we need this, to reduce three lines of code?

object-assign
-------------

[README](https://www.npmjs.com/package/object-assign) says:
> Node.js 4 and up, as well as every evergreen browser (Chrome, Edge, Firefox, Opera, Safari), support Object.assign() 🎉. If you target only those environments, then by all means, use Object.assign() instead of this package.

### CODE REVIEW

**Comment by voischev** [comment]
hi👋 
sry posthtml should support nodejs > v0.10 https://github.com/posthtml/posthtml/blob/master/package.json#L21

**Comment by hibiyasleep** [comment]
it should work now

File: index.js [hunk 1]
```diff
@@ -120,7 +119,10 @@ function parserWrapper() {
         return postHTMLParser(html, opt);
     }
 
-    if (arguments.length === 1 && isObject(arguments[0])) {
+    if (arguments.length === 1 &&
+        arguments[0] !== null &&
+        typeof arguments[0] === 'object' &&
+        Array.isArray(arguments[0]) === false) {
```
Comment (GitScrum) [inline index.js]: now it's hard to read

**Comment by GitScrum** [comment]
@hibiyasleep we appreciate your contribution to the development of the posthtml system, but this PR makes more incomprehensibility when reading, although productive.

File: index.js (hunk 1 above)
Comment (GitScrum) [reply index.js]: maybe so ?

```js
if (
  arguments.length === 1 &&  
  Boolean(arguments[0]) && 
  arguments[0].constructor.name === 'Object'
)
```

**Comment by GitScrum** [comment]
@hibiyasleep Thank you for your contribution, he is very important to us. We will be glad to receive any suggestions to the wishes and new PR 👍

**Pull request merged**